tests/test_integrals.py

```diff
@@ -1233,12 +1247,12 @@ def drift_analytic(data):
 
         # Exclude singularity not captured by analytic approximation for pitch near
         # the maximum |B|. (This is captured by the numerical integration).
-        pitch = get_pitch(np.min(B), np.max(B), 100)[:-1]
-        k2 = 0.5 * ((1 - B0 / pitch) / (epsilon * B0 / pitch) + 1)
+        pitch_inv = get_pitch_inv(np.min(B), np.max(B), 100).squeeze()[:-1]
+        k2 = 0.5 * ((1 - B0 / pitch_inv) / (epsilon * B0 / pitch_inv) + 1)
```

Comment:
should this be `B0 * pitch_inv`

a few other places around here as well

Reply:
The quantity $\lambda B$ is dimensionless, so this is corrected as is. What you are noticing is an artifact of commit 5653376, where I redefined `pitch` = $1 / \lambda$ for #1228 . Currently we define `pitch`=$\lambda$ and `pitch_inv`=$1\lambda$.